apps/server/src/modules/tool/controller/tool.controller.ts

```diff
@@ -27,4 +31,13 @@ export class ToolController {
 		const mapped: Lti11LaunchResponse = this.lti11ResponseMapper.mapAuthorizationToResponse(authorization);
```

Comment:
Please remove hier type after const => Type inference

Reply:
see above and here even more critically